bbc/simorgh
# WSTEAM1-676: Add fallback for default image

Resolves JIRA [WSTEAM1-676](https://jira.dev.bbc.co.uk/browse/WSTEAM1-676)

Overall changes
======
Add a fallback image for frosted promos: 
![Screenshot 2023-11-01 at 16 39 58](https://github.com/bbc/simorgh/assets/32307964/a8fb9731-36b9-4e8c-92f5-6c7069f9ce23)

Code changes
======
- _Adds optional chaining to `OptimoPromoFormatter` in `src/app/components/FrostedGlassPromo/withData.tsx`._
- _Adds default image props to `buildImageProperties` in `src/app/components/FrostedGlassPromo/withData.tsx`._

Testing
======
1. Turn on faulty 'informate' section for `https://www.test.bbc.com/pidgin/articles/cv6yjn1z042o`.
2. Go to `https://www.test.bbc.com/pidgin/articles/cv6yjn1z042o`.
3. Make sure informate section is rendering correctly with 'video 2' showing the default BBC logo. 

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)
